imap/structure_test.go

```diff
@@ -168,3 +168,20 @@ func TestParseMessage_GODT_2513(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, parsed)
 }
+
+func Fuzz_NewParsedMessage(f *testing.F) {
+	in_seed1, err_1 := os.ReadFile(filepath.Join("testdata", "rfc822.eml"))
```

Comment:
Odd that that the linter didn't catch this, but can you change this to `camelCase`, e.g.: `inSeed1`